### PR TITLE
add version to table, make plotting function

### PR DESCRIPTION
**Pull request by jamiemkass**
I added a simple plotting function for the pkg table. I think it's a useful idea, and could possibly be used as a figure for the paper. Also added the pkg version to the table.

### CODE REVIEW

File: R/plot_table.R [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#' @return Plot package characteristics from the sdmverse table using ggplot.
+#'
+#' @examples
+#' \dontrun{
+#' plot_table()
+#' }
+#'
+#' @export
+#'
+plot_table <- function(type = "tile") {
```
Comment (sylvainschmitt) [inline R/plot_table.R:20]: Very nice! Just unsure if the name could be more specific but I like it.

File: R/prep_table.R [hunk 2]
```diff
@@ -17,8 +17,8 @@ NULL
 #' @export
 #'
 prep_table <- function() {
-  version <- author <- cran <- github <- description <- NULL
+  author <- cran <- github <- description <- NULL
   list_packages() %>%
-    select(-version, -author, -cran, -github, -description) %>%
+    select(-author, -cran, -github, -description) %>%
```
Comment (sylvainschmitt) [inline R/prep_table.R:22]: Needed indeed.

**Review by sylvainschmitt — APPROVED** [review]

**Comment by sylvainschmitt** [comment]
The new function `plot_table` is not covered by automatic testing. Do you have time to add it or should I?

**Comment by jamiemkass** [comment]
I can add a test if you haven't already. I agree the name should be more specific. Should we just append "sdmverse_" to all functions?

**Pull request merged**